src/assets/content/dictionary/linux.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Linux
-definition: ''
-perspectives: []
+definition: 'linux is a unix-like operating system; that is open-source, tech agnostic, and community lead, driven and supported'
+perspectives: [Linus Torvalds developed the linux kernel is 1991. Some of the more common linux operating systems are: Ubuntu, Redhat, Fedora, Debian, Arch. Linux OS is used on the fastest top 500 supercomputers in the world]
```

Comment:
```suggestion
definition: Linux is a unix-like operating system; that is open-source, tech agnostic, and community lead, driven and supported. Linus Torvalds developed the linux kernel is 1991. Some of the more common linux operating systems are: Ubuntu, Redhat, Fedora, Debian, Arch. Linux OS is used on the fastest top 500 supercomputers in the world.
```
Bug: Since the second string is also part of the definition, we need to have them as one long line.

Reply:
hi @novellac thank you for the review, I agree with your amendment. 
may I ask, what would constitute a 'Perspective' comment ?